charts/spire/values.yaml

```diff
@@ -1,8 +1,16 @@
-# You can enable features that affect all services here.
-# global:
-#   telemetry:
-#     prometheus:
-#       enabled: true
+# You can enable config/features that affect all services here.
+global:
+  spire:
+    # -- Override all instances of clusterName
```

Comment:
Instead of `Override...` suggest just to use the description for the actual configurable. You can put a comment at the top of the section noting that the settings in this section apply globally and override stuff in subcharts.